conda_smithy/configure_feedstock.py

```diff
@@ -1124,6 +1124,8 @@ def render_drone(jinja_env, forge_config, forge_dir):
     )
 
 def render_README(jinja_env, forge_config, forge_dir):
+    if "README.md" in forge_config["skip_render"]:
```

Comment:
Please put a print statement in here saying that readme rendering is being skipped